selfdrive/car/hyundai/carcontroller.py

```diff
@@ -73,21 +73,32 @@ def update(self, CC, CS):
 
     if self.CP.carFingerprint in HDA2_CAR:
       # steering control
-      can_sends.append(hda2can.create_lkas(self.packer, CC.enabled, self.frame, CC.latActive, apply_steer))
+      if self.CP.carFingerprint in CAR.GENESIS_GV70:
+        can_sends.append(hda2can.create_lfa(self.packer, CC.enabled, self.frame, CC.latActive, apply_steer))
+      else:
+        can_sends.append(hda2can.create_lkas(self.packer, CC.enabled, self.frame, CC.latActive, apply_steer))
 
       if self.frame % 5 == 0:
         can_sends.append(hda2can.create_cam_0x2a4(self.packer, self.frame, CS.cam_0x2a4))
 
       # cruise cancel
       if (self.frame - self.last_button_frame) * DT_CTRL > 0.25:
+        #TODO: Improve this logic. Don't like bringing 
+        # the bus logic out here, but it's probably
+        # better than creating a new function
+        if self.CP.carFingerprint in CAR.GENESIS_GV70:
```

Comment:
You can, we have one for Honda: https://github.com/commaai/openpilot/blob/master/selfdrive/car/honda/hondacan.py#L11